shared/provision/password/container.tsx

```diff
@@ -18,9 +16,7 @@ const mapStateToProps = state => ({
 
 const mapDispatchToProps = dispatch => ({
   _onForgotPassword: username =>
-    flags.resetPipeline
-      ? dispatch(RecoverPasswordGen.createStartRecoverPassword({abortProvisioning: true, username}))
-      : dispatch(LoginGen.createLaunchForgotPasswordWebPage()),
```

Comment:
if we don't need this action anymore, can you delete it?

Reply:
Yeah thanks, done